lib/jobs/ansible-job.js

```diff
@@ -107,11 +97,27 @@ function ansibleJobFactory(
                 logger.warning("Result file not found. " + err.message);
                 self._done();
             } else {
-                self._done(err);
+                loadResultFile(self.outFileName,self.context)
```

Comment:
I think you might want to return here: 
`return loadResultFile(self.outFileName,self.context);`